src/ert/_c_wrappers/enkf/ensemble_config.py

```diff
@@ -441,9 +428,9 @@ def get_field_node(
             max_ = float(max_)
         return Field(
             name=name,
-            nx=grid.nx,
-            ny=grid.ny,
-            nz=grid.nz,
+            nx=int(dimensions[0]),
```

Comment:
Have you considered using a `NamedTuple` here instead of a `Tuple`?
We could then for example write `dimension.nx` instead of `dimension[0]` which I think reads better.

Reply:
added a NamedTuple named Shape